.github/workflows/deploy.yml

```
@@ -0,0 +1,520 @@
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 10
+      
+      - name: Find previous deployment
+        id: previous
+        run: |
+          # Get the previous successful deployment
+          PREVIOUS_SHA=$(git rev-list --max-count=2 HEAD | tail -n 1)
+          echo "sha=$PREVIOUS_SHA" >> $GITHUB_OUTPUT
+          echo "Rolling back to commit: $PREVIOUS_SHA"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)